app/api/services.py

```diff
@@ -0,0 +1,96 @@
+__author__ = 'Manuel Escriche'
+
+import logging
+from flask import jsonify, request, Response
+from functools import wraps
+from kconfig import deliveryBook, trackersBook, enablersBook, helpdeskNodesBook, chaptersBook
+from kernel.DataBoard import Data
+from kernel.NM_Aggregates import EnablerDeck, DevBacklog
+from kernel.NM_HelpDeskReporter import DeckReporter
+from kernel.NM_BacklogReporter import BacklogReporter
+from . import api
+
+
+master_token = "adc842f790105942e095f5f77484ffa8242a2ec3"
```

Comment:
A master_token here and a few lines below there is an username="api" --- These pieces of information shouldn't be here, should it?